docker_ci_deploy/tests/test_main.py

```diff
@@ -301,11 +301,10 @@ def test_failed_run_obfuscates_password(self, tmpdir):
         exit_1.chmod(exit_1.stat().mode | stat.S_IEXEC)
 
         runner = DockerCiDeployRunner(executable=str(exit_1))
-        with pytest.raises(CalledProcessError) as e_info:
+        with ExpectedException(CalledProcessError,
+                               After(str, Not(MatchesRegex(r'pa\$\$word')))):
```

Comment:
Is there a particular reason to use `$` rather than `5` or something here? It could make the regex a little simpler.